cqrs-user-query-service/src/test/java/dev/amir/userquery/framework/input/rabbitmq/mapper/UserMessageMapperTest.java

```diff
@@ -60,7 +61,6 @@ void test_Convert_WhenActiveFieldIsNotPresent_SetFalseByDefault() {
         assertEquals(message.name(), actual.getName());
         assertEquals(message.lastname(), actual.getLastname());
         assertEquals(message.email(), actual.getEmail());
-        assertFalse(actual.isActive());
```

Comment:
Read the purpose of this test and modify the logic if needed.
The current changes does not make sense